src/routes/tickets/util/getMetadataFromTicket.ts

```diff
@@ -1,11 +1,29 @@
 import { ITicket, ITicketMetaData } from '../controllers/ticketController';
 
+
 export const getMetadataFromTicket = (data: ITicket[]): ITicketMetaData[] => {
-	const metadata = data;
+  const metadata = data;
+  let flattenedMessage = '';
+
+  // Take all the messages put them together into a single string and if the string is equals or grater than 50 return. Once 50 add three dots at the end of te string and return that as previewString
+
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  metadata[0].messages[0].body.content.forEach(message => {
+    message.content?.forEach(content => {
+      if (flattenedMessage.length >= 50) return;
+      if (content.type === 'text' && typeof content.text === 'string') {
+        flattenedMessage = `${flattenedMessage} ${content.text}`;
```

Comment:
```suggestion
        flattenedMessage += " " + content.text;
```

Reply:
Any specific reason for making this change?

Reply:
It's just more clear IMO, and I think in @mrjvs's opinion too

Reply:
Fair enough 👍

Reply:
1. its easier to read. it litterally does + " " + msg. there is no need for templating.
  - if you want you can also do `msg = msg.concat(" ", otherMsg);`
2. its so much faster
<img src="https://i.imgur.com/qV0ROP5.png" />

Reply:
Will go with the concat route then.